### ISSUE TITLE
Small bug in the Analysis.Rmd vignette

### ISSUE THREAD

**Issue by maelle** [vpnagraj/hei#18]
It lacks the header

``` 
output: rmarkdown::html_vignette
vignette: >
  %\VignetteEngine{knitr::knitr}
  %\VignetteIndexEntry{hei Vignette}
  %\usepackage[UTF-8]{inputenc}
``` 

This issue is part of a [JOSS review](https://github.com/openjournals/joss-reviews/issues/417).

**Comment by maelle** [comment]
I noticed this because it wasn't built when using `devtools::build_vignettes`; instead, the function mentioned the missing header.

**Comment by vpnagraj** [comment]
thanks @maelle ! 
fixed as of 8b29340
